Sources/SwiftGRPCNIO/GRPCClientChannelHandler.swift

```diff
@@ -206,3 +220,12 @@ extension GRPCClientChannelHandler {
     context.close(mode: .all, promise: nil)
   }
 }
+
+/// Client user evenets.
+///
+/// - cancelled: The call has been cancelled.
```

Comment:
nit: document each enum case in-line? (Not sure if it makes a difference)

Reply:
Sure, Xcode defaults to putting them above like this (with "⌥⌘/") but inline is a little easier to read.